youtube_dl/extractor/funimation.py

```diff
@@ -215,9 +245,12 @@ def _real_extract(self, url):
         self._sort_formats(formats)
 
         title = item['title']
-        artist = item.get('artist')
-        if artist:
-            title = '%s - %s' % (artist, title)
```

Comment:
I'm not sure how I fooled myself, but I could've _swore_ that the "artist" key was returned". Not sure why I thought that. I'll revert the renaming.